src/System.Device.Gpio.Tests/README.md

```diff
@@ -0,0 +1,37 @@
+# How to run tests on the Raspberry Pi
```

Comment:
```suggestion
# How to run tests
```

I think this is not RPi specific

Reply:
Maybe stupid question, but which other platforms are supported? You need hardware with a GPIO header.

Reply:
anything ARM32 (v7 I believe) / ARM64. I.e. Hummingboard and there is plenty other but they are not as popular as RPi